Changed url routing for index.

debanjangg opened the pull request:
Changed landing page from https://<domain:port>/FunctionApp/index to https://<domain:port> to save a few seconds.

Aqina commented:
what??/

On Mon, Dec 9, 2024 at 6:26 PM netlify[bot] ***@***.***>
wrote:

> ❌ Deploy Preview for *frolicking-mochi-1062bc* failed.
> Name Link
> 🔨 Latest commit b6d26e6
> <https://github.com/Aqina/To_Do_List/commit/b6d26e6e5b7cda5d6d2081fe0a13406cf08b9986>
> 🔍 Latest deploy log
> https://app.netlify.com/sites/frolicking-mochi-1062bc/deploys/6756e89ec31fca00088b424c
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Aqina/To_Do_List/pull/1#issuecomment-2527853910>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQMVT2QNT4N7IRQ2INHT3232EWHQPAVCNFSM6AAAAABTIYVFQ2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKMRXHA2TGOJRGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

debanjangg commented:
Modified URL Routings to change the default index/landing page.

Example: Currently, you have to visit `http://127.0.0.1:8000/FunctionApp/index` to reach the index.
I changed that to `127.0.0.1:8000`, by changing both `myapp/urls.py` and `FunctionApp/urls.py`. So just running the server and clicking on the link will get you there.

[idk about the build fails, ig it's not set up properly or sth else]
